frontend/src/components/activity/content/LAThematicArea.vue

```diff
@@ -13,7 +13,6 @@
       :disabled="layoutMode || disabled"
       :menu-props="{
         maxWidth: 'min(290px, calc(100vw - 32px))',
-        offsetY: true,
```

Comment:
It looks far better now for me.
The saving animation is not visible anymore, but that's ok